photon-client/src/stores/StateStore.ts

```diff
@@ -95,12 +100,11 @@ export const useStateStore = defineStore("state", {
         clients: data.clients
       };
     },
-    updatePipelineResultsFromWebsocket(data: WebsocketPipelineResultUpdate) {
-      for (const cameraIndex in data) {
-        if (parseInt(cameraIndex) === this.currentCameraIndex) {
-          this.pipelineResults = data[cameraIndex];
-        }
-      }
+    updateBackendResultsFromWebsocket(data: WebsocketPipelineResultUpdate) {
+      this.backendResults = {
+        ...this.backendResults,
+        ...data
+      };
```

Comment:
This can be replaced by an Object.assign call.

Reply:
How would I do that exactly?